### ISSUE TITLE
Nixnote and Evernote insert different html code for bulleted list

### ISSUE THREAD

**Issue by ovid360** [robert7/nixnote2#147]
The html code for the bulleted lists in Nixnote and Evernote is different. Editing text containing bulleted lists in Nixnote and Evernote online has an unexpected behaviour. 

In online Evernote I make a simple bulleted list like this: 
![image](https://user-images.githubusercontent.com/60563095/73605024-f93a8780-45d3-11ea-863b-f080bb86acc5.png)
html code: `<ul><li><div>line 1</div></li></ul>`

Then I add a new line in online Evernote like this: 
![image](https://user-images.githubusercontent.com/60563095/73605015-d314e780-45d3-11ea-9b20-8414f295b6da.png)
html code: `<ul><li><div>line 1</div></li><li><div>line 2</div></li></ul>`

If I add the new line in Nixnote then it looks like this: 
![image](https://user-images.githubusercontent.com/60563095/73604284-09993500-45c9-11ea-9e73-d7d04f2d5d36.png)
html code: `<ul><li><div>line 1</div><div>line 2</div></li></ul>` (does not add an extra list item)

If I create the bulleted list only in Nixnote then it looks like this:
![image](https://user-images.githubusercontent.com/60563095/73604905-90064480-45d2-11ea-8412-6bf7a9cff725.png)
html code: `<ul><li>line 1</li><li>line 2</li></ul>` (missing `<div>` tags compared to Evernote)

So it looks that Evernote is adding `<li>` tags for a new line, whereas Nixnote doesn't. Also, Nixnote creates the lists with no `<div>` tags. 

**Specifications**
  - Nixnote Version: 202001302002-2.1.5-ubuntu18.04.1
  - Evernote online

**Comment by robert7** [comment]
I'm aware of the problem, but unfortunately this can't be fixed that easily. I can change the markup, which is inserted on "bulleted list" (but that doesn't help), but I can't (not easily) change the markup, which is after pressing enter on end of bulleted list.

The problem is old HTML editor (qt webkit) which is obsolete/unsupported for long time already... but unfortunately this isn't that easily to be replaced and is 100% out of my scope.

**Comment by ovid360** [comment]
OK, I understand now, thanks for the clarification

**Comment by easiestripes** [comment]
are there any suggested workarounds to this? basically all my notes utilize lists so not being able to add new list items to an existing list is a massive painpoint

**Comment by llabusch93** [comment]
Hello @robert7 

you said that the migration to a recent HTML editor is out of your scope. Who's scope is that and is he / she aware of the issue?
Is NixNote2 still in active development, or do you let this fade out, as the official Linux client was announced by Evernote (although it seems to take still a long time until we see a first release).

I like NixNote2, but the differences between the official Evernote clients are in some cases a problem.

**Comment by robert7** [comment]
@llabusch93 Sorry I don't have any time currently for the project and I don't use it any more (as I'm migrated do macOS).
Anyone who is interested, can continue, but currently none does, as far I know..

**Comment by boo-yee** [comment]
I find this issue recently, and I want to update some information in case some people are still troubled by it.

I tested it with the web Evernote, and I found that the web Evernote didn't output the <div> tag any more when a bullet list was inserted. but it still has a problem, that is, the first item in the bullet list is always excluded from the list after this note is saved. It might be a bug by Evernote, and they might fix it someday.

Now, we can use the bullet list in Nixnote normally except the problem above, but when pressing enter, a new bullet can be auto added.
